code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -133,9 +106,9 @@
 			HTML += "<br>"
 
 	HTML += "</body></html>"
-	user << browse(jointext(HTML,null), "window=assembly-\[REF(src)];size=600x350;border=1;can_resize=1;can_close=1;can_minimize=1")
+	user << browse(HTML, "window=assembly-[REF(src)];size=600x350;border=1;can_resize=1;can_close=1;can_minimize=1")
```

Comment:
Join is actually better way than adding these directly.

Reply:
Why? As far as I know, nothing else uses jointext for this purpose. It's a consistency change.

Reply:
Join only adds the result string to the string table, `+=` will also add all intermediate strings

Reply:
Nothing uses ```jointext()```, that's correct, they instead use ```list.Join()``` since string-builder UIs like this should be done with lists, not string concatenation.